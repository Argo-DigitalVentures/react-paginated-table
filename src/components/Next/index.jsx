import React from 'react';

function Next(props) {
  return (
    <button {...props}>Next</button>
  );
}

export default Next;
