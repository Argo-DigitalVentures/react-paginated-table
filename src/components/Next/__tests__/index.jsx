import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Next from '../';

describe('<Next />', () => {
  it('should accept className, style, and other DOM props', () => {
    const tree = shallow(
      <Next
        className="test"
        style={{
          color: 'green',
        }}
        aria-label="next page button"
      />,
    );
    expect(toJson(tree)).toMatchSnapshot();
  });
});
