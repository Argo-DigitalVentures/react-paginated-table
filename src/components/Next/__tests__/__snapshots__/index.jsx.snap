exports[`<Next /> should accept className, style, and other DOM props 1`] = `
<button
  aria-label="next page button"
  className="test"
  style={
    Object {
      "color": "green",
    }
  }>
  Next
</button>
`;
